StortSpelProjekt/Engine/src/Renderer/Renderer.cpp

```diff
@@ -673,6 +674,8 @@ void Renderer::InitGUI2DComponent(component::GUI2DComponent* component)
 		// Finally store the quad in m_pRenderer so it will be drawn
 		m_QuadComponents.push_back(component);
 	}
+
+	setRenderTasksGUI2DComponents();
```

Comment:
There is a function where this fits better. If you have 100 gui elements, you will set them 100 times for the render task.

Reply:
You are right, this function should not be here. It must be a leftover I forgot to remove